src/WinRT.Runtime/Configuration/FeatureSwitches.cs

```diff
@@ -77,4 +77,44 @@ private static bool GetConfigurationValue(string propertyName, ref int cachedRes
 
         return isEnabled;
     }
+
+    private static Projections.WuxMuxMode _wuxMuxMode = Projections.WuxMuxMode.Unknown;
+
+    internal static Projections.WuxMuxMode WuxMuxMode => GetWuMuxMode("CsWinRT.WuxMuxMode", ref _wuxMuxMode);
```

Comment:
So, thinking about this some more, I think we can simplify it to not even need any new enum types nor new feature switch helper methods compared to what we had before. We should also follow the same conventions as the other feature switches. Here's how I think we should structure this:
- We add this switch: `CSWINRT_USE_WINDOWS_UI_XAML_PROJECTIONS`
- The MSBuild property will be: `CsWinRTUseWindowsUIXamlProjections` (defaults to `false`)
- Here in `FeatureSwitches`, we just need the `UseWindowsUIXamlProjections` property (a `bool`)
- Everywhere else we simply check and branch on that flag
- We can have a single throw path for below .NET 5, in `Projections.cs` (we already have it anyway)

Reply:
We need the tri-state in MSBuild so we can detect in the SDK targets that you're trying to use the WUX projections but explicitly set MUX mode. We can use a boolean property for the feature switch though.